### PR TITLE
Add Lima KubeCon 22 EU talk to repo

### PR DESCRIPTION
**Pull request by jandubois**
Signed-off-by: Jan Dubois <jan.dubois@suse.com>

### CODE REVIEW

File: docs/talks/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Talks
+
+## Kubecon + CloudNativeCon Europe 2022
+
+[@AkihiroSuda](https://github.com/AkihiroSuda) and [@jandubois](https://github.com/jandubois) presented "Running containerd and k3s on macOS" about Lima.
+
+> It has been very hard to use Mac for developing containerized apps. A typical way is to use Docker for Mac, but it is not FLOSS. Another option is to install Docker and/or Kubernetes into VirtualBox, often via minikube, but it doesn't propagate localhost ports, and VirtualBox also doesn't support the ARM architecture. This session will show how to run containerd and k3s on macOS, using Lima and Rancher Desktop. Lima wraps QEMU in a simple CLI, with neat features for container users, such as filesystem sharing and automatic localhost port forwarding, as well as DNS and proxy propagation for enterprise networks. Rancher Desktop wraps Lima with k3s integration and GUI.
+
+Read the [slides](kubecon-2022-eu.pdf) or watch the [video](https://www.youtube.com/watch?v=g5GCsbjkzRM).
```
Comment (AkihiroSuda) [inline docs/talks/README.md:9]: I'm not sure we want to add PDF blobs into the git repo.
Can we just link https://static.sched.com/hosted_files/kccnceu2022/5f/lima.pdf ?

Comment (jandubois) [reply docs/talks/README.md:9]: Yes, that is fine. I wasn't sure if that was going to be a perma-link, but I guess we can use it while it works. Just need to make sure we keep a copy around if the site ever deletes old assets.

File: docs/talks/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Talks
```
Comment (AkihiroSuda) [inline docs/talks/README.md:1]: > docs/talks/README.md

Can be moved to docs/talks.md ?

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**